foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/OpenAiAssistantMessage.java

```diff
@@ -15,23 +20,37 @@
 @Beta
 @Value
 @Accessors(fluent = true)
-class OpenAiAssistantMessage implements OpenAiMessage {
+@AllArgsConstructor(access = PACKAGE)
+public class OpenAiAssistantMessage implements OpenAiMessage {
 
-  /** The role of the message. */
+  /** The role associated with this message. */
   @Nonnull String role = "assistant";
 
   /** The content of the message. */
-  @Nonnull String content;
+  @Getter(onMethod_ = @Beta)
+  @Nonnull
+  OpenAiMessageContent content;
```

Comment:
content should be nullable, also tool_calls is missing
[See AssistantMessage](https://github.com/SAP/ai-sdk-java/blob/9cc853f49cf9c39d64da21c70a01fa21fb8c2a0a/orchestration/src/main/java/com/sap/ai/sdk/orchestration/AssistantMessage.java#L56)